src/history.ts

```diff
@@ -1,6 +1,8 @@
 import * as vscode from 'vscode'
 
 import { CommandDescriptor, CommandState, InputKind } from './commands'
+import { OffsetRange, OffsetSelection, OffsetEdgeTransformationBehaviour } from './utils/offsetSelection'
```

Comment:
Please order imports alphabetically (in this line and by swapping with the next line).